frontend/docs/pages/home/dags.mdx

```diff
@@ -78,16 +61,7 @@ For instance, `def task_1(foo: EmptyModel, bar: Context) -> None:` is perfectly
 </Tabs.Tab>
 <Tabs.Tab title="Typescript">
 
-```typescript
-const toLower = simple.task({
-  name: "to-lower",
-  fn: (input) => {
-    return {
-      TransformedMessage: input.Message.toLowerCase(),
-    };
-  },
-});
-```
+<Snippet src={snippets.typescript.simple.workflow.declaring_a_task} />
```

Comment:
This looks like a standalone task snippet, not a task that's part of a workflow?